google/cloud/internal/async_rpc_details.h

```diff
@@ -73,7 +74,8 @@ class AsyncUnaryRpcFuture : public AsyncGrpcOperation {
     if (!ok) {
       // `Finish()` always returns `true` for unary RPCs, so the only time we
       // get `!ok` is after `Shutdown()` was called; treat that as "cancelled".
-      promise_.set_value(Status(StatusCode::kCancelled, "call cancelled"));
+      promise_.set_value(internal::CancelledError("gRPC async unary RPC failed",
```

Comment:
Why? The original message is more accurate?

Reply:
I thought the original message was ambiguous. I can revert to that.

Reply:
Reverted the message.